### PR TITLE
Release v0.8.1 version

### PR DESCRIPTION
**Pull request by ChugunovRoman**

### CODE REVIEW

**Review by jannuary — CHANGES_REQUESTED** [review]
You have two closing </ul> tags in the appdata file. (line 41)

File: io.github.Figma_Linux.figma_linux.appdata.xml [hunk 1]
```diff
@@ -1,60 +1,45 @@
...
-          <li>other: Improve application name in desktop file. (#161)</li>
-          <li>other: Bump Electron.js version to 12.0.2</li>
-          <li>other: add arm64v8</li>
-          <li>other: added generate_release_notes.pl script</li>
-          <li>other: added script for bump figma linux version</li>
+          <li>Bug Fixes:</li>
+          <ul>
+            <li>fix size of close icon</li>
+            <li>fix build app for ppa repository</li>
+          </ul>
```
Comment (jannuary) [inline io.github.Figma_Linux.figma_linux.appdata.xml]: Furthermore, you are not supposed to remove the older changelogs from appdata.

**Review by jannuary — APPROVED** [review]

**Comment by gasinvein** [comment]
So, should this be merged (as asked in https://github.com/flathub/io.github.Figma_Linux.figma_linux/commit/e3a51bd76e9f3ad2677c781eff20c055fae700dd#r54139261)?

**Comment by ChugunovRoman** [comment]
> So, should this be merged (as asked in [e3a51bd#r54139261](https://github.com/flathub/io.github.Figma_Linux.figma_linux/commit/e3a51bd76e9f3ad2677c781eff20c055fae700dd#r54139261))?

Yes, it's need to merge

**Comment by gasinvein** [comment]
Oh wait, I don't see a merge button here either. Something isn't right.

@bilelmoussaoui @TingPing Any idea what's wrong here?

**Comment by nedrichards** [comment]
@ChugunovRoman you need to accept the github invitation to have write access. I've sent it again.

**Comment by ChugunovRoman** [comment]
> @ChugunovRoman you need to accept the github invitation to have write access. I've sent it again.

@nedrichards Hi, I accepted the github invitation, but I don't see the merge button still. I cannot merge https://github.com/flathub/io.github.Figma_Linux.figma_linux/pull/8

**Comment by gasinvein** [comment]
@ChugunovRoman I don't see a "Collaborator" badge on your comments headers either. Weird.

**Pull request merged**